x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/ml/pipeline_inference.yml

```diff
@@ -99,6 +99,12 @@ setup:
           { "product": "VCR", "cost": 180, "time": 1587501233000 }
           { "index": {} }
           { "product": "Laptop", "cost": 15000, "time": 1587501233000 }
+  - do:
+      headers:
+        Authorization: "Basic eF9wYWNrX3Jlc3RfdXNlcjp4LXBhY2stdGVzdC1wYXNzd29yZA==" # run as x_pack_rest_user, i.e. the test setup superuser
+      cluster.health:
+        index: [".ml-inference-*", ".ml-stats-*"]
+        wait_for_status: green
```

Comment:
There was another separate test failure where these tests failed because `.ml-inference-000001` shards were not initialized. Seems weird as the earlier PUT worked. But adding this to make sure.